fplll/bkz.cpp

```diff
@@ -559,7 +559,8 @@ template <class ZT, class FT> bool BKZReduction<ZT, FT>::bkz()
     print_params(param, cerr);
     cerr << endl;
   }
-  cputime_start = cputime();
+
+  cputime_start = (flags & BKZ_DUMP_GSO) ? cputime() : 0;
```

Comment:
I think this should also use `BKZ_MAX_TIME`:

```c++
if ((flags & BKZ_MAX_TIME) && (cputime() - cputime_start) * 0.001 >= param.max_time)
```

Reply:
Good catch: I've added ``` flags & BKZ_MAX_TIME``` to the ternary.